sbnanaobj/StandardRecord/SRSlice.cxx

```diff
@@ -13,9 +13,15 @@ namespace caf
 {
   FlashMatch::FlashMatch():
     present(false),
-    score(std::numeric_limits<float>::signaling_NaN()),
     time(std::numeric_limits<float>::signaling_NaN()),
-    pe(std::numeric_limits<float>::signaling_NaN())
+    charge_q(std::numeric_limits<float>::signaling_NaN()),
+    light_pe(std::numeric_limits<float>::signaling_NaN()),
+    score(std::numeric_limits<float>::signaling_NaN()),
+    scr_y(std::numeric_limits<float>::signaling_NaN()),
+    scr_z(std::numeric_limits<float>::signaling_NaN()),
+    scr_rr(std::numeric_limits<float>::signaling_NaN()),
+    scr_ratio(std::numeric_limits<float>::signaling_NaN())
+    // what about chargeXYZ and lightXYZ?
```

Comment:
Looks like this is pre-existing, but why are there two almost-identical FlashMatch classes? Can we merge them, or otherwise rationalize this?

Reply:
This is a question for @FernandaPsihas. I see there are three `fmatch` getting stored on the cafs. So maybe it's related to that?

Reply:
I vote for removing the `caf::FlashMatch` class. It must be a holdover from past days.

Reply:
That sounds like a good idea. I converted everything I could see. I don't see this directly used anywhere, only the `SRFlashMatch` version

Reply:
On slack Fernanda agrees. Latest commit removes the duplicated class.